src/argo/workflow.yaml

```diff
@@ -154,8 +190,9 @@ spec:
           - /state/{{inputs.parameters.local-path}}
           - gs://{{inputs.parameters.gcs-bucket}}/{{inputs.parameters.gcs-path}}
         volumeMounts:
-          - name: state-dir
+          - name: {{inputs.parameters.volume-name}}
             mountPath: /state
+            readOnly: true
```

Comment:
We should update this line to account for `/state` now being read-only: https://github.com/DataBiosphere/clinvar-ingest/pull/10/files#diff-f8e60a2d7352da7b045ba3259d61ae03R184. We can either remove that setting entirely, or figure out how to use ephemeral state in these workflows.

Reply:
Not sure I follow; don't we want to set the readOnly flag to `true` here to make sure we mount it as readOnly so that we can use the ReadOnlyMany access mode to concurrently read from it for the xml upload step and the xml-to-json step?